src/game/client/components/nameplates.cpp

```diff
@@ -201,12 +201,26 @@ void CNamePlates::RenderNameplatePos(vec2 Position, const CNetObj_PlayerInfo *pP
 				TextRender()->Text(0, Position.x - PosArmor / 2.0f, YOffset, AFontSize, aArmor, -1);
 			}
 		}
+	}
 
-		TextRender()->TextColor(TextRender()->DefaultTextColor());
-		TextRender()->TextOutlineColor(TextRender()->DefaultTextOutlineColor());
-
-		TextRender()->SetRenderFlags(0);
+	if(g_Config.m_Debug)
+	{
+		CCharacter *pCharacter = m_pClient->m_GameWorld.GetCharacterByID(pPlayerInfo->m_ClientID);
+		if(pCharacter)
+		{
+			YOffset -= FontSize;
+			char aBuf[128];
+			str_format(aBuf, sizeof(aBuf), "%d", m_pClient->m_GameWorld.GetCharacterByID(pPlayerInfo->m_ClientID)->GetStrongWeakID());
```

Comment:
```suggestion
			str_format(aBuf, sizeof(aBuf), "%d", pCharacter->GetStrongWeakID());
```

You already have the character

Reply:
Done